services/workflow.go

```diff
@@ -41,6 +52,87 @@ func (workflow Workflow) Call(params ...interface{}) (result interface{}, err er
 
 func ld_ld_sync(bytes []byte) error {
 	log.Println("Executing")
-	log.Println(string(bytes))
+
+	// Unmarshal the JSON byte array into the map
+	var jsonBody map[string]interface{}
+	err := json.Unmarshal(bytes, &jsonBody)
+	if err != nil {
+		slog.Error(err.Error())
+		return err
+	}
+
+	if jsonBody["LeadStatus"] == "Referred Out" && jsonBody["LeadSubstatus"] == "Pending Review" && jsonBody["LeadReferredTo"] == "The Capital Law Firm" {
+		// There should be more error handling so that when an error is raised we should try to send the request later, right now
+		// these messages are also offset
+		sendLeadToClf(jsonBody)
+	}
+
+	return nil
+}
+
+// Should move to utils later
+
+func GetOrDefaultString(m map[string]interface{}, key string, defaultStr string) string {
+	if value, ok := m[key]; ok {
+		if str, ok := value.(string); ok {
+			return str
+		}
+	}
+	return defaultStr
+}
+
+func GetOrDefaultInt(m map[string]int, key string, defaultInt int) int {
+	if num, ok := m[key]; ok {
+		return num
+	}
+	return defaultInt
+}
+
+func sendLeadToClf(jsonBody map[string]interface{}) error {
+	defaultStr := ""
+	sinkUrl := "http://localhost:5000"
```

Comment:
Cool, let's use environment variables to load this information in.

See main for how we load environment variables:
https://github.com/integrandio/integrand/blob/master/main.go

Then to use, grab it using `os.Getenv("YOUR_VARIABLE")`